src/behaviors/random-channel.js

```diff
@@ -0,0 +1,36 @@
+import Behavior from './behavior.js';
+
+class RandomChannel extends Behavior {
+  constructor(settings = {}) {
+    settings.name = settings.name || 'Random Channel';
+
+    super(settings);
+  }
+
+  initialize(bot) {
+    super.initialize(bot);
+
+    let channels = bot._api('channels.list', { token: bot.token, exclude_archived: 1 });
+
+    channels.then(function(data) {
+      let randomChannel = data.channels[Math.floor(Math.random() * data.channels.length)];
```

Comment:
You should do this as a do/while loop, because what happens if, on the second try, the channel is either all-staff or announcements. It *could* happen.